xrviz/control.py

```diff
@@ -11,24 +11,29 @@
 
 class Control(SigSlot):
     """
-    This section allows the user to control the other subsections,
-    such as displayer, fields.
+    This section arranges all the panes in the form of tabs,
+    hence providing the controls to inteact with all the
+    panes.
 
     Parameters
     ----------
-    data: `xarray` instance: `DataSet` or `DataArray`
-           datset is used to initialize.
+    data: `xr.core.dataarray.DataWithCoords`
```

Comment:
Prefer previous, simple description, `xarray.DataSet` or `xarray.DataArray`
You can then specify in the description what that means. I don't expect users to know what DataWithCoords is.